### PR TITLE
Create Atm

### PR DESCRIPTION
**Pull request by b1zman**

### CODE REVIEW

File: hw4_epam_lab/src/test/java/com/atw/AtwTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        Atm cardAtm = new Atm(card);
+        cardAtm.withdrawAtm(30);
+        assertEquals(card.getAccountBalance(), 20, 0.0000001);
+    }
+
+    @Test
+    void cardBalance_ShowBalanceCard() {
+        Card card = new DebetCard(50, "Bob");
+        Atm cardAtm = new Atm(card);
+        cardAtm.cardBalance();
```
Comment (kislyakdmitry) [inline hw4_epam_lab/src/test/java/com/atw/AtwTest.java]: Does it make any sense?
It has to be like this:
`assertEquals(cardAtm.cardBalance(), 50, 0.0000001);`
And line 23 has to be removed

File: hw4_epam_lab/src/test/java/com/atw/AtwTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+
+    @Test
+    void convertAtw_ShowConvertBalanceCard_WhenConvert() {
+        Card card = new DebetCard(5, "Bob");
+        Atm cardAtm = new Atm(card);
+        cardAtm.convertAtm(10);
+        assertEquals(card.getAccountBalance(), 50, 0.00000001);
+    }
```
Comment (kislyakdmitry) [inline hw4_epam_lab/src/test/java/com/atw/AtwTest.java]: Please, add a negative case:
When we use debit card and withdraw value more than the card has.

**Review by kislyakdmitry — CHANGES_REQUESTED** [review]

File: hw4_epam_lab/src/test/java/com/atw/AtwTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
+package com.atw;
+
+import com.atw.card.Card;
+import com.atw.card.DebetCard;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AtwTest {
```
Comment (Jericho-d) [inline hw4_epam_lab/src/test/java/com/atw/AtwTest.java]: What is the Atw?)

File: hw4_epam_lab/src/test/java/com/atw/AtwTest.java (hunk 1 above)
Comment (b1zman) [reply hw4_epam_lab/src/test/java/com/atw/AtwTest.java]: Done

File: hw4_epam_lab/src/test/java/com/atw/AtwTest.java (hunk 2 above)
Comment (b1zman) [reply hw4_epam_lab/src/test/java/com/atw/AtwTest.java]: Done

File: hw4_epam_lab/src/test/java/com/atw/AtwTest.java (hunk 3 above)
Comment (b1zman) [reply hw4_epam_lab/src/test/java/com/atw/AtwTest.java]: Done

**Pull request merged**